addons/web/static/src/core/utils/draggable_hook_builder.js

```diff
@@ -107,10 +107,7 @@ const elCache = {};
  * @returns {string}
  */
 function camelToKebab(str) {
-    return str.replaceAll(
-        /(?<=.)[A-Z]|[A-Z]/g,
-        (match, prev) => `${prev ? "-" : ""}${match.toLowerCase()}`
-    );
+    return str.replace(/([a-z])([A-Z])/g, "$1-$2").toLowerCase();
```

Comment:
To replicate the same result as the old RegEx
```suggestion
    return str
        .split("")
        .map(
            (element, index) =>
                `${index > 0 && element.match(/[A-Z]/) ? "-" : ""}${element.toLowerCase()}`
        )
        .join("");
```

Reply:
But do we want to exactly replicate the same behavior (thinking about the multi-uppercase-letters).

A few examples:

```
fooBarBuz => Original: 'foo-bar-buz' New: 'foo-bar-buz'
foo => Original: 'foo' New: 'foo'
FooBar => Original: 'foo-bar' New: 'foo-bar'
foo bar => Original: 'foo bar' New: 'foo bar'
foo123bar => Original: 'foo123bar' New: 'foo123bar'
foo123Bar => Original: 'foo123-bar' New: 'foo123bar'
fooBARBuz => Original: 'foo-b-a-r-buz' New: 'foo-barbuz'
```

But what do we expect for the last one ?

Reply:
I did not try, but it seems to me that the new regexp does not work as you say for these cases:
```
FooBar => Original: 'foo-bar' New: 'Foo-bar'
fooBARBuz => Original: 'foo-b-a-r-buz' New: 'foo-bARBuz'
```